scripts/build_pkgdown.R

```diff
@@ -18,15 +20,37 @@ if (requireNamespace("PEcAn.logger", quietly = TRUE)) {
   }
 }
 
-logger("Building pkgdown docs for:", paste(packages, collapse = ", "))
+pkg_config <- function() {
 
+  list(
+    url = "https://pecanproject.github.io/",
```

Comment:
I found a partial answer to my question above about the difference between `URL:` in the DESCRIPTION and `url` here: When we specify a url here, _all internal links_ are changed from relative paths to absolute URLS.

It's possible that's desirable for some cases, but for our local Docker deployment it means we'll either have locally hosted pages that are filled with links out to the Github-hosted equivalents, or that we'll need to rewrite the url field when compiling pkgdocs at Docker build time.

I think for this PR my recommendation is to remove this url from the yml so that the local pages keep working locally; we could rethink this later as needed.
```suggestion
```